.gitignore

```diff
@@ -12,10 +12,18 @@ schema1.5/
 
 # Compiled python modules.
 *.pyc
+build/
 
 # Setuptools distribution folder.
 /dist/
 
 # Python egg metadata, regenerated from source files by setuptools.
 /*.egg-info
 /*.egg
+
+# ghdl compiler
+work*.cf
+
+# created by the test, but not under git
+example.pdf
```

Comment:
It is considered bad practise in general to version control generated files, HOWEVER, the benefit for new users to be able to see what can be generated, by looking at github.com in the browser, I feel in this case outweighs any disadvantages. Please put them back. I don't mind them being regenerated a lot

Reply:
Hi Andreas,

Thanks for responding.
I see your point and will send a patch soon.
Later, a slightly better solution could be to create a documentation website on github.io.